framework/source/class/qx/data/SingleValueBinding.js

```diff
@@ -495,6 +495,9 @@ qx.Class.define("qx.data.SingleValueBinding",
               target = target["get" + firstUpPropName]();
             }
             targets[j] = target;
+            if (!target) {
+              break;
```

Comment:
I'm not sure of the full context of what's going on here, but is `break` the appropriate thing to do, or is `continue`? It's inside a loop of various property names. Do we want, upon finding one of them to not exist, to stop handling the others?

Reply:
Surely `break` is correct - this loop is re-establishing the listeners, going from left to right, so for `foo.bar.baz` and where `foo.bar` is null, you have to break because it makes no sense to subsequently try and bind to `.baz`

Reply:
Now I understand the context here, and yes, of course.